targets/trusty/debian/changelog

```diff
@@ -0,0 +1,95 @@
+openhim-mediator-mhero (0.1.0-15~trusty) trusty; urgency=medium
+
+  * Release Debian Build 0.1.0-15.
+
+ -- Martin Brocker <martin@jembi.org>  Tue, 03 May 2016 14:30:30 +0200
+
+openhim-mediator-mhero (0.1.0-14~trusty) trusty; urgency=medium
+
+  * Release Debian Build 0.1.0-14.
+
+ -- Martin Brocker <martin@jembi.org>  Tue, 03 May 2016 13:57:23 +0200
+
+openhim-mediator-mhero (0.1.0-13~trusty) trusty; urgency=medium
+
+  * Release Debian Build 0.1.0-13.
+
+ -- Martin Brocker <martin@jembi.org>  Tue, 03 May 2016 12:04:03 +0200
+
+openhim-mediator-mhero (0.1.0-12~trusty) trusty; urgency=medium
+
+  * Release Debian Build 0.1.0-12.
+
+ -- Martin Brocker <martin@jembi.org>  Tue, 03 May 2016 11:12:55 +0200
+
+openhim-mediator-mhero (0.1.0-11~trusty) trusty; urgency=medium
+
+  * Release Debian Build 0.1.0-11.
+
+ -- Martin Brocker <martin@jembi.org>  Tue, 03 May 2016 10:44:12 +0200
+
+openhim-mediator-mhero (0.1.0-10~trusty) trusty; urgency=medium
+
+  * Release Debian Build 0.1.0-10.
+
+ -- Martin Brocker <martin@jembi.org>  Tue, 03 May 2016 10:37:47 +0200
+
+openhim-mediator-mhero (0.1.0-9~trusty) trusty; urgency=medium
+
+  * Release Debian Build 0.1.0-9.
+
+ -- Martin Brocker <martin@jembi.org>  Tue, 03 May 2016 10:08:00 +0200
+
+openhim-mediator-mhero (0.1.0-8~trusty) trusty; urgency=medium
+
+  * Release Debian Build 0.1.0-8.
+
+ -- Martin Brocker <martin@jembi.org>  Tue, 03 May 2016 10:00:02 +0200
+
+openhim-mediator-mhero (0.1.0-7~trusty) trusty; urgency=medium
+
+  * Release Debian Build 0.1.0-7.
+
+ -- Martin Brocker <martin@jembi.org>  Tue, 03 May 2016 09:47:06 +0200
+
+openhim-mediator-mhero (0.1.0-6~trusty) trusty; urgency=medium
+
+  * Release Debian Build 0.1.0-6.
+
+ -- Martin Brocker <martin@jembi.org>  Tue, 03 May 2016 09:42:25 +0200
+
+openhim-mediator-mhero (0.1.0-5~trusty) trusty; urgency=medium
+
+  * Release Debian Build 0.1.0-5.
+
+ -- Martin Brocker <martin@jembi.org>  Tue, 03 May 2016 09:27:54 +0200
+
+openhim-mediator-mhero (0.1.0-4~trusty) trusty; urgency=medium
+
+  * Release Debian Build 0.1.0-4.
+
+ -- Martin Brocker <martin@jembi.org>  Tue, 03 May 2016 09:11:42 +0200
+
+openhim-mediator-mhero (0.1.0-3~trusty) trusty; urgency=medium
+
+  * Release Debian Build 0.1.0-3.
+
+ -- Martin Brocker <martin@jembi.org>  Fri, 29 Apr 2016 15:49:23 +0200
+
+openhim-mediator-mhero (0.1.0-2~trusty) trusty; urgency=medium
+
+  * Release Debian Build 0.1.0-2.
+
+ -- Martin Brocker <martin@jembi.org>  Fri, 29 Apr 2016 15:35:23 +0200
+
+openhim-mediator-mhero (0.1.0-1~trusty) trusty; urgency=medium
+
+  * Release Debian Build 0.1.0-1.
+
+ -- Martin Brocker <martin@jembi.org>  Fri, 29 Apr 2016 15:33:24 +0200
+
+openhim-mediator-mhero (0.1.0-1~trusty) trusty; urgency=low
```

Comment:
We need to reset this back to the first entry as most of these builds are just testing builds. We should only commit the changelog when we upload the package.